fr

```diff
@@ -63,6 +63,7 @@ merdeuse
 merdeux
 meuf
 nègre
+negro
```

Comment:
Cella est un anglicisme et le mot exists déjà dans la list d'anglais https://github.com/LDNOOBW/List-of-Dirty-Naughty-Obscene-and-Otherwise-Bad-Words/blob/master/en#L233 

Mais dans ce cas ci, Cela est assez commun en français que ça fait du sense de l'ajouter au français aussi.